python/ql/test/experimental/meta/ConceptsTest.qll

```diff
@@ -255,52 +255,64 @@ module HttpServerRequestHandlerTest implements TestSig {
   }
 }
 
-class HttpServerHttpResponseTest extends InlineExpectationsTest {
-  File file;
+abstract class DedicatedResponseTest extends string {
```

Comment:
Note that this moves part of the functionality that first occurred in a number of individual tests to this shared file. I decided not to introduce another parameterized module that then needs to be instantiated, as that would affect many test.